### PR TITLE
Removed unused code.

### PR DESCRIPTION
**Pull request by brason**
I think that should be just about all of it. Just hope I haven't deleted too much...

### CODE REVIEW

File: src/objects/SkinnedMesh.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ THREE.SkinnedMesh.prototype.normalizeSkinWeights = function () {
 
 };
 
-THREE.SkinnedMesh.prototype.updateMatrixWorld = function( force ) {
+THREE.SkinnedMesh.prototype.updateMatrixWorld = function() {
 
 	THREE.Mesh.prototype.updateMatrixWorld.call( this, true );
```
Comment (WestLangley) [inline src/objects/SkinnedMesh.js]: I expect it should be

    THREE.Mesh.prototype.updateMatrixWorld.call( this, force );

I wonder if this change will cause a problem?...

File: src/Three.js [hunk 2]
```diff
@@ -199,21 +199,21 @@ THREE.Projector = function () {
 
 	THREE.error( 'THREE.Projector has been moved to /examples/js/renderers/Projector.js.' );
 
-	this.projectVector = function ( vector, camera ) {
+	this.projectVector = function () {
 
 		THREE.warn( 'THREE.Projector: .projectVector() is now vector.project().' );
 		vector.project( camera );
```
Comment (mrdoob) [inline src/Three.js]: `vector` and `camera` are referenced here...

File: src/Three.js [hunk 3]
```diff
@@ -199,21 +199,21 @@ THREE.Projector = function () {
...
 
 	};
 
-	this.unprojectVector = function ( vector, camera ) {
+	this.unprojectVector = function () {
 
 		THREE.warn( 'THREE.Projector: .unprojectVector() is now vector.unproject().' );
 		vector.unproject( camera );
```
Comment (mrdoob) [inline src/Three.js]: `vector` and `camera` are referenced here too...

**Comment by mrdoob** [comment]
Hmmm... this is introducing a bunch of bugs...
I can't merge this but I'll use it as reference to clean up code by hand.

**Comment by brason** [comment]
Okay, guess I deleted too much. I'm not sure how to rebase yet, so I created a new pull request, hope that is ok.

**Pull request closed without merging**